src/components/CourseDetails.js

```diff
@@ -60,41 +49,41 @@ export default function CourseDetails(props) {
       <div aria-labelledby={'openbutton' + course.crn}>
         <Button
           className={classes.button}
-          color="secondary"
+          color='secondary'
           onClick={() => setOpen(true)}
           id={'openbutton' + course.crn}
-          aria-label="course description"
+          aria-label='course description'
         >
-          {t('description')}
+          Description
```

Comment:
This is a much bigger scope than this PR, but a fun upcoming task for anyone who wants it is going to be getting translations back into the portal in a more reliable way